src/routes/users/handlers.js

```diff
@@ -21,3 +21,23 @@ export async function getPossibleCollaborators(req, res) {
     await handleErrors(req, res, error, { namespace: 'SERVICE:USER' });
   }
 }
+
+export async function getPossibleStateCodes(req, res) {
+  try {
+    const user = await userById(req.session.userId);
+    const regions = user.permissions.map((permission) => permission.regionId);
+    const grants = await Grant.findAll({
```

Comment:
Can you move this to a method in the `services/grant.js` file? There are probably instances of us calling the DB from handlers but it is nice to keep DB calls inside services and outside of handlers